pkg/cloudprovider/aws/apis/v1alpha1/provider_validation.go

```diff
@@ -115,6 +117,16 @@ func (a *AWS) validateTags() (errs *apis.FieldError) {
 	return errs
 }
 
+func (a *AWS) validateContext() (errs *apis.FieldError) {
```

Comment:
I think we can remove this validation and assume the user will provide a string if they're using it. If not, I think the logs will sufficiently convey an error.